packages/design-system-react-native/src/primitives/Blockies/Blockies.tsx

```diff
@@ -7,13 +7,13 @@ import { toDataUrl } from './Blockies.utilities';
 
 import type { BlockiesProps } from './Blockies.types';
 
-const Blockies = ({ address, size = 32, ...imageProps }: BlockiesProps) => {
+const Blockies = ({ address, size = 32, ...props }: BlockiesProps) => {
   return (
     <Image
       source={{ uri: toDataUrl(address) }}
       width={size}
       height={size}
-      {...imageProps}
+      {...props}
```

Comment:
Suggestion: I realize we use `imageProps` for the `<img>` tag in other components when passing image attributes, but generally, for spreading props to the root element, we can use `...props`. This won’t be visible to the end user.

![Screenshot 2025-03-18 at 11 14 30 AM](https://github.com/user-attachments/assets/57756b45-af5b-4a65-b408-d0d3222577f5)